MASTER3in1_v2025.08.05_v0.2.py

```
@@ -0,0 +1,1222 @@
+            self.word_output_frame.pack_forget()
+
+    def toggle_all_offices(self, select):
+        for var in self.office_vars.values():
+            var.set(select)
+
+    def load_data(self):
+        filepath = filedialog.askopenfilename(
+            title="Select Data Digest",
+            filetypes=[("Excel files", "*.xlsx *.xlsm"), ("All files", "*.*")]
+        )
+        if not filepath: return
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)